process-terminate.js

```diff
@@ -0,0 +1,27 @@
+module.exports = function (RED) {
+    function ProcessTerminate(config) {
+        RED.nodes.createNode(this, config);
+        var node = this;
+
+        node.on('input', function (msg) {
+            const engine = RED.nodes.getNode(config.engine);
+            const client = engine.engineClient;
+
+            if (!client) {
+                node.error('No engine configured.');
+                return;
+            }
+
+            client.processInstances
+                .terminateProcessInstance(msg.payload, engine.identity)
```

Comment:
in der Hilfe steht, es muss `processInstanceId` als Input vorhanden sein. Was ist denn hier korrekt?